cfme/containers/replicator.py

```diff
@@ -42,6 +45,19 @@ def get_detail(self, *ident):
         self.load_details(refresh=True)
         return details_page.infoblock.text(*ident)
 
+    @classmethod
+    def get_random_instances(cls, provider, count=1, appliance=None):
```

Comment:
well, I see this redundant to use helper since in every object the mgmt.list_* is different and also the instantiation arguments. also there are some objects (like image) that this function is implemented differently.